Template/Tag/TawkToTag.php

```diff
@@ -35,6 +35,12 @@ public function getParameters() {
                 $field->validators[] = new NotEmpty();
                 $field->validators[] = new CharacterLength(16, 30); // we limit to 30 so users don't accidentally enter a 32 digit API key
             }),
+            $this->makeSetting('tawkToWidgetId', '', FieldConfig::TYPE_STRING, function (FieldConfig $field) {
```

Comment:
fyi I've just checked in an old account we had, there it was still `default`. So to not break it for existing users, I assume setting `default` as default value might help. Otherwise we risk that it worn't work anymore for customers that have their account already for a while?

I then created another new widget in our account, and there it did use two different parts indeed. The second part was 9 characters long. Allowing up to 20 sounds good.

Reply:
I have updated it to default and reduced the minLength limit to 7 otherwise it will fail as default is of 7 characters.